ui/app/components/transform-edit-form.js

```diff
@@ -0,0 +1,20 @@
+import Component from '@ember/component';
+import { inject as service } from '@ember/service';
+
+export default Component.extend({
+  init() {
+    this._super(...arguments);
+    // TODO: don't need confirm?
+    this.set('backendType', 'transform');
+  },
+  store: service(),
+  actions: {
+    // TODO modify the parameters and potentially rename
+    createOrUpdate(type, event) {
```

Comment:
you could rename this to `handleCreateOrUpdate` if you want to be clear that this action is an event handler calling the method on the adapter. in ember we see a lot of `handleFoo` where `foo` is either the event (submit, enter) or an action passed down from above.

Reply:
oh, good idea.